programs/main.cpp

```diff
@@ -328,7 +328,11 @@ struct Checker
     {
         checkRequiredInstructions();
     }
-} checker __attribute__((init_priority(101)));  /// Run before other static initializers.
+} checker
+#ifndef __APPLE__
```

Comment:
But it works perfectly in our cross builds.
Does it mean we cannot just check by `__APPLE__` and we need something more sophisticated?

Reply:
Ideally, `__has_attribute(init_priority)` should do the thing. But it didn't work, it was true, but the problem persisted.